packages/bpk-component-modal/src/BpkModalDialog-test.js

```diff
@@ -46,6 +46,7 @@ describe('BpkModalDialog', () => {
           closeEvents={closeEvents}
           dialogRef={jest.fn()}
           isIphone={false}
+          fullScreenOnMobile
```

Comment:
why add these? they have a default value

Reply:
Ah apologies it doesn't have a default value. Ignore me!

Reply:
They have default value for the modal component, not for the modal dialog component